### ISSUE TITLE
Failed to activate the script package

### ISSUE THREAD

**Issue by jkatzer** [rgbkrk/atom-script#396]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 0.194.0 :warning: **in 1.0 API Preview Mode** :warning:
**System**: Mac OS X 10.10.3
**Thrown From**: [script](https://github.com/rgbkrk/atom-script) package, v2.19.0

### Stack Trace

Failed to activate the script package

```
At Cannot read property 'prependToTop' of undefined

TypeError: Cannot read property 'prependToTop' of undefined
  at ScriptOptionsView.module.exports.ScriptOptionsView.initialize (/Users/jason/.atom/packages/script/lib/script-options-view.coffee:51:23)
  at ScriptOptionsView.View (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/node_modules/space-pen/lib/space-pen.js:184:25)
  at new ScriptOptionsView (/Users/jason/.atom/packages/script/lib/script-options-view.coffee:6:3)
  at Object.module.exports.activate (/Users/jason/.atom/packages/script/lib/script.coffee:28:30)
  at Package.module.exports.Package.activateNow (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:222:19)
  at /Applications/Atom.app/Contents/Resources/app.asar/src/package.js:203:30
  at Package.module.exports.Package.measure (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:147:15)
  at Package.module.exports.Package.activate (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:195:14)
  at PackageManager.module.exports.PackageManager.activatePackage (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:434:21)
  at /Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:418:29
  at Config.module.exports.Config.transact (/Applications/Atom.app/Contents/Resources/app.asar/src/config.js:311:16)
  at PackageManager.module.exports.PackageManager.activatePackages (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:413:19)
  at PackageManager.module.exports.PackageManager.activate (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:394:46)
  at Atom.module.exports.Atom.startEditorWindow (/Applications/Atom.app/Contents/Resources/app.asar/src/atom.js:623:21)
  at Object.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/window-bootstrap.js:12:8)
  at Object.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/window-bootstrap.js:23:4)
  at Module._compile (module.js:452:26)
  at Object.loadFile [as .js] (/Applications/Atom.app/Contents/Resources/app.asar/src/babel.js:162:21)
  at Module.load (module.js:347:32)
  at Function.Module._load (module.js:302:12)
  at Module.require (module.js:357:17)
  at require (module.js:376:17)
  at window.onload (file:///Applications/Atom.app/Contents/Resources/app.asar/static/index.js:54:25)

```

**Comment by jigga** [comment]
I have the exact same setup and what follows the exact same problem. Below's mine stack trace (a bit different).

```
TypeError: Cannot read property 'prependToTop' of undefined
    at ScriptOptionsView.module.exports.ScriptOptionsView.initialize (/Users/jigga/.atom/packages/script/lib/script-options-view.coffee:115:25)
    at ScriptOptionsView.View (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/node_modules/space-pen/lib/space-pen.js:184:25)
    at new ScriptOptionsView (/Users/jigga/.atom/packages/script/lib/script-options-view.coffee:12:54)
    at Object.module.exports.activate (/Users/jigga/.atom/packages/script/lib/script.coffee:36:32)
    at Package.module.exports.Package.activateNow (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:222:19)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/package.js:203:30
    at Package.module.exports.Package.measure (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:147:15)
    at Package.module.exports.Package.activate (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:195:14)
    at PackageManager.module.exports.PackageManager.activatePackage (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:434:21)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:418:29
```

**Comment by erran** [comment]
Duplicate of https://github.com/rgbkrk/atom-script/issues/395. I'll leave this issue open since it picked up more discussion and they'll both be closed by #407.

**Comment by AlexMckey** [comment]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.1.0-beta1
**System**: Unknown Windows Version
**Thrown From**: [script](https://github.com/rgbkrk/atom-script) package, v3.0.1


### Stack Trace

Failed to activate the script package

```
At Cannot find module 'underscore'

Error: Cannot find module 'underscore'
    at Module._resolveFilename (module.js:334:15)
    at Function.Module._resolveFilename (C:\Users\Александр\AppData\Local\atom\app-1.1.0-beta1\resources\app.asar\src\module-cache.js:383:52)
    at Function.Module._load (module.js:284:25)
    at Module.require (module.js:363:17)
    at require (module.js:382:17)
    at Object.<anonymous> (C:\Users\Александр\.atom\packages\script\lib\grammars.coffee:4:7)
    at Object.<anonymous> (C:\Users\Александр\.atom\packages\script\lib\grammars.coffee:812:4)
    at Module._compile (module.js:428:26)
    at Object.defineProperty.value [as .coffee] (C:\Users\Александр\AppData\Local\atom\app-1.1.0-beta1\resources\app.asar\src\compile-cache.js:169:21)
    at Module.load (module.js:353:32)
```

### Commands

```
     -0:02.9.0 script:run (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {}
}
```

### Installed Packages

```coffee
# User
autocomplete-haskell, v0.5.1
haskell-ghc-mod, v0.9.13
haskell-pointfree, v0.1.2
language-haskell, v1.4.11
minimap, v4.14.0
script, v3.0.1

# Dev
No dev packages
```
